widget/embedded/src/components/NotificationContent/NotificationContent.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Notification } from '../../types/notification';
+
+import {
+  ChainToken,
+  ChevronRightIcon,
+  ListItemButton,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useMetaStore } from '../../store/meta';
+import { useNotificationStore } from '../../store/notification';
+
+import { Container, Images, List } from './NotificationContent.styles';
+
+export function NotificationContent() {
+  const MAX_NOTIFICATIONS_DISPLAYED = 4;
+  const navigate = useNavigate();
+  const notifications: Notification[] =
+    useNotificationStore.use.notifications();
+
+  const setAsRead = useNotificationStore.use.setAsRead();
+
+  const { tokens, blockchains } = useMetaStore.use.meta();
+  const sortedNotification = notifications
+    .sort((a, b) => {
+      const POSITIVE = 1;
+      const NEGATIVE = -1;
+      if (a.read === b.read) {
+        return a.creationTime < b.creationTime ? POSITIVE : NEGATIVE;
+      }
+      return a.read ? POSITIVE : NEGATIVE;
+    })
+    .slice(0, MAX_NOTIFICATIONS_DISPLAYED);
+
+  const handleOnClick = (requestId: Notification['requestId']) => {
+    setAsRead(requestId);
+    navigate(`/swaps/${requestId}`);
+  };
+
+  return (
+    <Container>
+      <List>
```

Comment:
![image](https://github.com/rango-exchange/rango-client/assets/120931880/5214753c-f9cd-47e4-a6bb-685d626331e3)
I think it would be better to hide the notification list if it's empty (or disable the button)